Chart.yaml

```diff
@@ -1,7 +1,7 @@
 # apiVersion v1 is Helm 2
 apiVersion: v1
 name: airflow
-version: 0.18.0
+version: 0.18.1
```

Comment:
I don't think 0.18.0 has actually been released yet (but I'm not 100% sure) -- we're still testing changes for Airflow 2.0 here.